app/models/waste_exemptions_engine/transient_registration.rb

```diff
@@ -37,15 +34,8 @@ def registration_attributes
           "temp_grid_reference", "temp_site_description",
           "address_finder_error", "transient_addresses",
           "transient_registration_exemptions", "transient_people",
-          "created_at", "updated_at"
+          "created_at", "updated_at", "type"
```

Comment:
Wow! This is a quite long list of attributes! :D Would the other way around generate less code (`take` the ones we need rather than `except` the one we don't)? Might also be the case to externalise this list in some constant for better visibility and easier changes

Reply:
Good idea – moved to a constant in https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/6ad41c6c1b5620cd14c1376afc15a2036686bcaa

I think the list of attributes to `take` is slightly longer than the `except` ones, so have left it this way around for now.